actors/builtin/miner/miner_actor.go

```diff
@@ -445,6 +445,8 @@ func (a Actor) PreCommitSector(rt Runtime, params *SectorPreCommitInfo) *adt.Emp
 	var st State
 	newlyVested := big.Zero()
 	rt.State().Transaction(&st, func() {
+		verifyPledgeMeetsInitialRequirements(rt, &st)
```

Comment:
This method name is getting dusty. How about `verifyBalanceCoversPledge`?

Reply:
Agree it could use a new name.  It's going to need to change again in the upcoming fee debt pr so I'll put off the the change until that one.